pages/common/pio-ci.md

```diff
@@ -0,0 +1,29 @@
+# pio ci
+
+> Build PlatformIO projects with arbitrary source code structure.
+> This will create a new temporary project, which the source code will be copied into.
```

Comment:
Perhaps this will read better?
```suggestion
> Build PlatformIO projects with an arbitrary source code structure.
> This will create a new temporary project which the source code will be copied into.
```

By `new temporary project`, do you mean a `new temporary directory`?

Reply:
It creates a new project within the `/tmp` directory